teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    @specialization = specialization
+    super
```

Comment:
- When inheriting a parent class, please make sure that the class constructor extends to the parent's constructor. Please read this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#inheritance) for more information